rfcs/QueryLevelNullability.md

```diff
@@ -126,16 +192,25 @@ query {
 ```
 
 ## ✅ RFC Goals
+
 - Non-nullable syntax that is based off of syntax that developers will already be familiar with
+- Enable GraphQL codegen tools to generate more ergonomic types
 
 ## 🚫 RFC Non-goals
 
 ## 🗳️ Alternatives considered
 
-### Use a directive such as `@nonNull` instead
-This is the alternative being used at some of the companies represented in this proposal for the time being.
-However this feels like a common enough need to call for a language feature. A single language feature
-rather than using directives as a workaround would enable more unified public tooling around GraphQL.
+### An official `@nonNull` directive
+
+This solution offers the same benefits as the proposed solution. Since many GraphQL codegen tools already support the `@skip` and `@include` directives, this solution likely has a faster turnaround.
+
+### Use a custom `@nonNull` directive
```

Comment:
There is another issue with a custom directive. Not only the code generators, but the cache implementations of GraphQL client libraries also need to understand the custom directive to behave correctly. 

Currently, when a client library caches a null field based on an operation without a directive, it will return the null field for another operation with this directive.

Reply:
Oh thanks for calling that out! I'll create a section for work items that'll result if this RFC is accepted. to track these kinds of things.